### DISCUSSION TITLE
倍速与倍速记忆分开 或者只记忆特定视频

### DISCUSSION THREAD

**Discussion by LyntNy4n** [the1812/Bilibili-Evolved#3056] [功能建议 / Ideas]
我希望有自定义的倍速 在教程类的视频里使用. 但是因为记忆的原因 在其他普通视频也会用该倍速播放
由于打开普通视频的频率远高于教程类视频 所以我反而希望不记住倍速播放
或者记住特定视频的倍速就好了

**Comment by Geeyun-JY3** [comment]
请善用搜索：#2746、https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.4-preview
